blockchain/common_test.go

```diff
@@ -56,55 +56,39 @@ func isSupportedDbType(dbType string) bool {
 	return false
 }
 
-// createTestDatabase creates a test database with the provided database name
-// and database type for the given network.
-func createTestDatabase(dbName string, dbType string, net wire.CurrencyNet) (database.DB, func(), error) {
+// createTestDatabase creates a test database with the provided database type
+// for the given network.
+func createTestDatabase(t testing.TB, dbType string, net wire.CurrencyNet) (database.DB, error) {
```

Comment:
This is removing the teardown func which also cleanly closes the database in addition to removing it.  That will lead to erroneous test failures on Windows due to inability to remove the temp dir due to it being in use, because, unlike Unix, you can't remove in-use things on Windows.